server/models/user/verifyAdminToken.js

```diff
@@ -9,6 +9,9 @@ function verifyToken(req, res, next) {
     if (err)
     return res.status(500).send({ auth: false, message: 'Failed to authenticate token.' });
     // if everything good, save to request for use in other routes
+    if (req.username != "admin" && req.password != "admin2020") {
```

Comment:
TLDR: Prefer `===` to `==` and `!==` to `!=`
Javascript attempts to coerce values if they are of different types and the rules by which it does are somehow complicated to memorize, things like `0 == ''    // evaluates to true`, which sometimes is not what you want.

You can read more [here](https://github.com/jamesallardice/jslint-error-explanations/blob/master/option-articles/eqeqeq-jshint.md)  when you get a minute